Replace autoFill with autofill for spelling consistency

rniwa opened the pull request:
#### 7a4de99c0908b404b8bf48c7fc602153b581fa20
<pre>
Replace autoFill with autofill for spelling consistency
<a href="https://bugs.webkit.org/show_bug.cgi?id=282553">https://bugs.webkit.org/show_bug.cgi?id=282553</a>

Reviewed by Anne van Kesteren.

Updated the spelling of HTMLInputElement&apos;s autofill member functions from autoFill to autofill for consistency.
Also removed &quot;is&quot; prefix from member functions as a preparation to expose them as JavaScript API in near future.

* LayoutTests/accessibility/auto-fill-crash.html:
* LayoutTests/accessibility/auto-fill-types.html:
* LayoutTests/accessibility/ios-simulator/strong-password-field.html:
* LayoutTests/accessibility/secure-field-value.html:
* LayoutTests/fast/forms/auto-fill-button/caps-lock-indicator-should-be-visible-after-hiding-auto-fill-strong-password-button.html:
* LayoutTests/fast/forms/auto-fill-button/caps-lock-indicator-should-not-be-visible-when-auto-fill-strong-password-button-is-visible-expected.html:
* LayoutTests/fast/forms/auto-fill-button/caps-lock-indicator-should-not-be-visible-when-auto-fill-strong-password-button-is-visible.html:
* LayoutTests/fast/forms/auto-fill-button/hide-auto-fill-button-when-input-becomes-disabled.html:
* LayoutTests/fast/forms/auto-fill-button/hide-auto-fill-button-when-input-becomes-readonly.html:
* LayoutTests/fast/forms/auto-fill-button/input-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-auto-filled-and-obscured.html:
* LayoutTests/fast/forms/auto-fill-button/input-contacts-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-credit-card-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-disabled-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-loading-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-readonly-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-readonly-non-empty-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-baseline-alignment.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-line-height-expected.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-line-height.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-taller-button-height.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable.html:
* LayoutTests/fast/forms/auto-fill-button/last-auto-fill-button-type-expected.txt:
* LayoutTests/fast/forms/auto-fill-button/last-auto-fill-button-type.html:
* LayoutTests/fast/forms/auto-fill-button/mouse-down-input-mouse-release-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/resources/process-auto-fill-button-type-and-invoke-runTest.js:
(window.onload):
* LayoutTests/fast/forms/auto-fill-button/show-auto-fill-button-while-selection-inside-field.html:
* LayoutTests/fast/forms/auto-fill-button/show-correct-auto-fill-button-when-auto-fill-button-type-changes-expected.html:
* LayoutTests/fast/forms/auto-fill-button/show-correct-auto-fill-button-when-auto-fill-button-type-changes.html:
* LayoutTests/fast/forms/input-autofilled-and-obscured-clear-field-programatically.html:
* LayoutTests/fast/forms/input-autofilled-and-obscured-clear-form-programatically.html:
* LayoutTests/fast/forms/input-autofilled-and-obscured.html:
* LayoutTests/fast/forms/input-autofilled-and-viewable.html:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
(WebCore::AccessibilityObject::isValueAutofillAvailable const):
(WebCore::AccessibilityObject::valueAutofillButtonType const):
* Source/WebCore/css/SelectorCheckerTestFunctions.h:
(WebCore::isAutofilled):
(WebCore::isAutofilledStrongPassword):
(WebCore::isAutofilledStrongPasswordViewable):
(WebCore::isAutofilledAndObscured):
* Source/WebCore/editing/TextManipulationController.cpp:
(WebCore::shouldIgnoreNodeInTextField):
* Source/WebCore/editing/VisibleSelection.cpp:
(WebCore::VisibleSelection::isInAutoFilledAndViewableField const):
* Source/WebCore/html/HTMLInputElement.cpp:
(WebCore::HTMLInputElement::resignStrongPasswordAppearance):
(WebCore::HTMLInputElement::reset):
(WebCore::HTMLInputElement::setValue):
(WebCore::HTMLInputElement::setValueFromRenderer):
(WebCore::HTMLInputElement::setAutofilled):
(WebCore::HTMLInputElement::setAutofilledAndViewable):
(WebCore::HTMLInputElement::setAutofilledAndObscured):
(WebCore::HTMLInputElement::setAutofillButtonType):
(WebCore::HTMLInputElement::createInnerTextStyle):
(WebCore::HTMLInputElement::setAutoFilled): Deleted.
(WebCore::HTMLInputElement::setAutoFilledAndViewable): Deleted.
(WebCore::HTMLInputElement::setAutoFilledAndObscured): Deleted.
(WebCore::HTMLInputElement::setShowAutoFillButton): Deleted.
* Source/WebCore/html/HTMLInputElement.h:
* Source/WebCore/html/TextFieldInputType.cpp:
(WebCore::TextFieldInputType::shouldDrawCapsLockIndicator const):
(WebCore::TextFieldInputType::shouldDrawAutoFillButton const):
(WebCore::TextFieldInputType::updateAutoFillButton):
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
(WebCore::isStrongPasswordTextField):
* Source/WebCore/rendering/RenderTextControlSingleLine.cpp:
(WebCore::RenderTextControlSingleLine::layout):
* Source/WebCore/rendering/RenderTextControlSingleLine.h:
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::setAutofilled):
(WebCore::Internals::setAutofilledAndViewable):
(WebCore::Internals::setAutofilledAndObscured):
(WebCore::toAutofillButtonType):
(WebCore::toInternalsAutofillButtonType):
(WebCore::Internals::setAutofillButtonType):
(WebCore::Internals::autofillButtonType):
(WebCore::Internals::lastAutofillButtonType):
(WebCore::Internals::setAutoFilledAndViewable): Deleted.
(WebCore::Internals::setAutoFilledAndObscured): Deleted.
(WebCore::toAutoFillButtonType): Deleted.
(WebCore::toInternalsAutoFillButtonType): Deleted.
(WebCore::Internals::setShowAutoFillButton): Deleted.
(WebCore::Internals::autoFillButtonType): Deleted.
(WebCore::Internals::lastAutoFillButtonType): Deleted.
* Source/WebCore/testing/Internals.h:
* Source/WebCore/testing/Internals.idl:
* Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebFormManager.cpp:
(webkit_web_form_manager_input_element_is_auto_filled):
* Source/WebKit/WebProcess/InjectedBundle/DOM/InjectedBundleNodeHandle.cpp:
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFilled const):
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFilledAndViewable const):
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFilledAndObscured const):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFilled):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFilledAndViewable):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFilledAndObscured):
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFillButtonEnabled const):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFillButtonEnabled):
(WebKit::InjectedBundleNodeHandle::htmlInputElementAutoFillButtonType const):
(WebKit::InjectedBundleNodeHandle::htmlInputElementLastAutoFillButtonType const):
(WebKit::InjectedBundleNodeHandle::isAutoFillAvailable const):
(WebKit::InjectedBundleNodeHandle::setAutoFillAvailable):
* Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm:
(WebKit::WebPage::requestDocumentEditingContext):
* Source/WebKitLegacy/mac/DOM/WebDOMOperations.mm:
(-[DOMHTMLInputElement _isAutofilled]):
(-[DOMHTMLInputElement _isAutoFilledAndViewable]):
(-[DOMHTMLInputElement _setAutofilled:]):
(-[DOMHTMLInputElement _setAutoFilledAndViewable:]):

Canonical link: <a href="https://commits.webkit.org/286185@main">https://commits.webkit.org/286185@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d1a0b48fd223f19220bfe3b20ce8b949b02854ad

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/75005 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/54435 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/27822 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/79443 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/26244 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/77122 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/63571 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/2220 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/58884 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/17156 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/78072 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/49034 "13 new passes 3 flakes 1 failures") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/64432 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/39270 "Passed tests") | 
| | [❌ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/46350 "Exiting early after 10 failures. 231 tests run. 1 flakes") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/21933 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/24576 "Built successfully") | 
| | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/67477 "Found 1 new API test failure: TestWebKitAPI.WebKit.CopyInAutoFilledAndViewablePasswordField (failure)") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/22275 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/80923 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/2323 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/1405 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/67135 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/2472 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/64451 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/66436 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/10375 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/8536 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/11588 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/2288 "Built successfully") | [  ~~🛠 mac-safer-cpp~~](https://ews-build.webkit.org/#/builders/120/builds/5156 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/2316 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/3237 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/2323 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
Looks good modulo Linux bots and some test failures that presumably need rebasing.

rniwa committed:
Replace autoFill with autofill for spelling consistency
https://bugs.webkit.org/show_bug.cgi?id=282553

Reviewed by Anne van Kesteren.

Updated the spelling of HTMLInputElement's autofill member functions from autoFill to autofill for consistency.
Also removed "is" prefix from member functions as a preparation to expose them as JavaScript API in near future.

* LayoutTests/accessibility/auto-fill-crash.html:
* LayoutTests/accessibility/auto-fill-types.html:
* LayoutTests/accessibility/ios-simulator/strong-password-field.html:
* LayoutTests/accessibility/secure-field-value.html:
* LayoutTests/fast/forms/auto-fill-button/caps-lock-indicator-should-be-visible-after-hiding-auto-fill-strong-password-button.html:
* LayoutTests/fast/forms/auto-fill-button/caps-lock-indicator-should-not-be-visible-when-auto-fill-strong-password-button-is-visible-expected.html:
* LayoutTests/fast/forms/auto-fill-button/caps-lock-indicator-should-not-be-visible-when-auto-fill-strong-password-button-is-visible.html:
* LayoutTests/fast/forms/auto-fill-button/hide-auto-fill-button-when-input-becomes-disabled.html:
* LayoutTests/fast/forms/auto-fill-button/hide-auto-fill-button-when-input-becomes-readonly.html:
* LayoutTests/fast/forms/auto-fill-button/input-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-auto-filled-and-obscured.html:
* LayoutTests/fast/forms/auto-fill-button/input-contacts-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-credit-card-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-disabled-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-loading-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-readonly-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-readonly-non-empty-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-baseline-alignment.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-line-height-expected.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-line-height.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable-taller-button-height.html:
* LayoutTests/fast/forms/auto-fill-button/input-strong-password-viewable.html:
* LayoutTests/fast/forms/auto-fill-button/last-auto-fill-button-type-expected.txt:
* LayoutTests/fast/forms/auto-fill-button/last-auto-fill-button-type.html:
* LayoutTests/fast/forms/auto-fill-button/mouse-down-input-mouse-release-auto-fill-button.html:
* LayoutTests/fast/forms/auto-fill-button/resources/process-auto-fill-button-type-and-invoke-runTest.js:
(window.onload):
* LayoutTests/fast/forms/auto-fill-button/show-auto-fill-button-while-selection-inside-field.html:
* LayoutTests/fast/forms/auto-fill-button/show-correct-auto-fill-button-when-auto-fill-button-type-changes-expected.html:
* LayoutTests/fast/forms/auto-fill-button/show-correct-auto-fill-button-when-auto-fill-button-type-changes.html:
* LayoutTests/fast/forms/input-autofilled-and-obscured-clear-field-programatically.html:
* LayoutTests/fast/forms/input-autofilled-and-obscured-clear-form-programatically.html:
* LayoutTests/fast/forms/input-autofilled-and-obscured.html:
* LayoutTests/fast/forms/input-autofilled-and-viewable.html:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
(WebCore::AccessibilityObject::isValueAutofillAvailable const):
(WebCore::AccessibilityObject::valueAutofillButtonType const):
* Source/WebCore/css/SelectorCheckerTestFunctions.h:
(WebCore::isAutofilled):
(WebCore::isAutofilledStrongPassword):
(WebCore::isAutofilledStrongPasswordViewable):
(WebCore::isAutofilledAndObscured):
* Source/WebCore/editing/TextManipulationController.cpp:
(WebCore::shouldIgnoreNodeInTextField):
* Source/WebCore/editing/VisibleSelection.cpp:
(WebCore::VisibleSelection::isInAutoFilledAndViewableField const):
* Source/WebCore/html/HTMLInputElement.cpp:
(WebCore::HTMLInputElement::resignStrongPasswordAppearance):
(WebCore::HTMLInputElement::reset):
(WebCore::HTMLInputElement::setValue):
(WebCore::HTMLInputElement::setValueFromRenderer):
(WebCore::HTMLInputElement::setAutofilled):
(WebCore::HTMLInputElement::setAutofilledAndViewable):
(WebCore::HTMLInputElement::setAutofilledAndObscured):
(WebCore::HTMLInputElement::setAutofillButtonType):
(WebCore::HTMLInputElement::createInnerTextStyle):
(WebCore::HTMLInputElement::setAutoFilled): Deleted.
(WebCore::HTMLInputElement::setAutoFilledAndViewable): Deleted.
(WebCore::HTMLInputElement::setAutoFilledAndObscured): Deleted.
(WebCore::HTMLInputElement::setShowAutoFillButton): Deleted.
* Source/WebCore/html/HTMLInputElement.h:
* Source/WebCore/html/TextFieldInputType.cpp:
(WebCore::TextFieldInputType::shouldDrawCapsLockIndicator const):
(WebCore::TextFieldInputType::shouldDrawAutoFillButton const):
(WebCore::TextFieldInputType::updateAutoFillButton):
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
(WebCore::isStrongPasswordTextField):
* Source/WebCore/rendering/RenderTextControlSingleLine.cpp:
(WebCore::RenderTextControlSingleLine::layout):
* Source/WebCore/rendering/RenderTextControlSingleLine.h:
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::setAutofilled):
(WebCore::Internals::setAutofilledAndViewable):
(WebCore::Internals::setAutofilledAndObscured):
(WebCore::toAutofillButtonType):
(WebCore::toInternalsAutofillButtonType):
(WebCore::Internals::setAutofillButtonType):
(WebCore::Internals::autofillButtonType):
(WebCore::Internals::lastAutofillButtonType):
(WebCore::Internals::setAutoFilledAndViewable): Deleted.
(WebCore::Internals::setAutoFilledAndObscured): Deleted.
(WebCore::toAutoFillButtonType): Deleted.
(WebCore::toInternalsAutoFillButtonType): Deleted.
(WebCore::Internals::setShowAutoFillButton): Deleted.
(WebCore::Internals::autoFillButtonType): Deleted.
(WebCore::Internals::lastAutoFillButtonType): Deleted.
* Source/WebCore/testing/Internals.h:
* Source/WebCore/testing/Internals.idl:
* Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebFormManager.cpp:
(webkit_web_form_manager_input_element_is_auto_filled):
* Source/WebKit/WebProcess/InjectedBundle/DOM/InjectedBundleNodeHandle.cpp:
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFilled const):
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFilledAndViewable const):
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFilledAndObscured const):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFilled):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFilledAndViewable):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFilledAndObscured):
(WebKit::InjectedBundleNodeHandle::isHTMLInputElementAutoFillButtonEnabled const):
(WebKit::InjectedBundleNodeHandle::setHTMLInputElementAutoFillButtonEnabled):
(WebKit::InjectedBundleNodeHandle::htmlInputElementAutoFillButtonType const):
(WebKit::InjectedBundleNodeHandle::htmlInputElementLastAutoFillButtonType const):
(WebKit::InjectedBundleNodeHandle::isAutoFillAvailable const):
(WebKit::InjectedBundleNodeHandle::setAutoFillAvailable):
* Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm:
(WebKit::WebPage::requestDocumentEditingContext):
* Source/WebKitLegacy/mac/DOM/WebDOMOperations.mm:
(-[DOMHTMLInputElement _isAutofilled]):
(-[DOMHTMLInputElement _isAutoFilledAndViewable]):
(-[DOMHTMLInputElement _setAutofilled:]):
(-[DOMHTMLInputElement _setAutoFilledAndViewable:]):

Canonical link: https://commits.webkit.org/286185@main
